docs/adr/0001-retiring-licence-finder.md

```diff
@@ -0,0 +1,82 @@
+
+When rendered, licence pages resemble transaction pages, and have an "Apply now" button that takes the user to one of two places: a licence managed by a competent authority on an external website (e.g. TV Licence) or a postcode lookup that allows users to apply for the licence via their local authority. These local authority licences are managed in [Licensify].
+
+Currently users can either find licences in the tool by narrowing down a list of "sectors" and "activities" or by performing a keyword search. Regardless of the approach, the list of results shown to the user is found by querying elastic search with the unique licence identifier assigned to each licence.
+
+If the licence is found in elastic search a link to the licence is displayed to the user.
+
+If the licence cannot be found in elastic search only the title of the licence is displayed to the user. This title may not match the title in the content item.
+
+### Problems with the current approach
+
+Only a few licences can be considered "mainstream". Most are for a specialist activity. However as licences are managed in Mainstream Publisher only GDS content designers can create licence pages.  As Content Designers across government cannot create licence pages, they have taken to using other Whitehall formats to create licence content. This means that the licences created in other formats do not have a licence identifier, and therefore do not appear in the licence-finder results.
+
+Another reason that not all licences appear in the licence finder results is because a developer is needed to updated the CSV files and run a rake task, a step that is easy to miss.
+
+It is possible for no results to be displayed to the user if they do a text search. For example, "restaurant" doesn't yield any results, even though it is in the name of a sector.
+
+Design-wise, the existing tool doesn't match any of the other design patterns on GOV.UK (e.g. smart-answers or finders), nor does it use any design system components.
+
+Finally, the existing tool doesn't contain any extra analytics tracking on the search field or the [sectors] and [activities] list making it difficult to track user behaviour.
+
+## Decision
+
+This Licence Finder application will be retired and replaced with a specialist finder similar to the [Air Accidents Investigation Branch reports] finder.
+
+Specialist Finders find documents with a particular special document schema and are configured in [Specialist Publisher].
+Specialist Finders are an existing pattern on GOV.UK and follow the design system. The specialist finder render already contains analytics tracking for the facets, search box and results clicked.
+
+Another benefit of this approach is that licences will still appear in the GOV.UK site search. Also, the default behaviour of finders is to show all of the available licences to the user when the finder loads, so the user will always see some results.
```

Comment:
We should probably add here that a user could also search for a licence by title (and description? Not sure the Elasticsearch fields queried), which is a vast improvement on the current tool which can only search by sector / activity... which doesn't even work properly 😄

Reply:
Case in point: restaurant 😱